components/BlurhashImage.vue

```diff
@@ -15,6 +15,7 @@
         class="absolute"
         :src="image"
         v-bind="$attrs"
+        :alt="item.Name"
```

Comment:
I don't think all of them should have a name.

For example, the background image on the home header is only illustrative in purpose. On the other hand, logos should have an alt for screen readers.